src/CMakeLists.txt

```diff
@@ -43,6 +43,16 @@ add_library(samplerate
 # ALIAS to use if libsamplerate is included from other project with add_subdirectory()
 add_library(SampleRate::samplerate ALIAS samplerate)
 
+if(MULTI_THREADING)
+  if(WIN32)
```

Comment:
I guess we need  to check `MSVC` variable. MinGW compiler probably will fail.

Reply:
commit: 7348582e57dcf5c0ee65ff104f413893648489a8